runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessAttendedVariables.ts

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from '#src-app/store';
+import { currentProcessSelector } from '#src-app/store/slices/Process';
+
+const useProcessAttendedVariables = () => {
+    const { executionInfo, isAttended } = useSelector(currentProcessSelector);
+    const { passedInVariables } = useSelector((state) => state.process.modeler);
+    
+    if (!isAttended) {
+        return [];
+    }
+
+    const inOutInfo = JSON.parse(executionInfo);    
+
+    const attendedVariables =
+        isAttended && executionInfo
+            ? [{name: passedInVariables[0], inOutInfo}, {name: passedInVariables[1], inOutInfo}]
```

Comment:
what is the type of `passedInVariables`? If its not a tuple like `[value, value]` then are we sure there are always 2 values?
I need more info because I don't get this code